cppapi/server/eventsupplier.h

```diff
@@ -247,6 +247,20 @@ public :
 	TANGO_IMP_EXP static ZmqEventSupplier *create(Util *);
 	virtual ~ZmqEventSupplier();
 
+
+    class Future
+    {
+    public:
+    	Future();
+    	void set();
```

Comment:
Please try to be consistent with the style of the codebase. We usually indent by either 4 spaces or tabs (depends on the surrounding code), but if I recall I've never seen 2-space indentation in cppTango.